subscriptions/configuration/subscription_templates.rst

```diff
@@ -0,0 +1,81 @@
+==========================
+Use subscription templates
+==========================
+
+The subscription business model is becoming more and more popular. Are you wondering why? Because a
+subscription model simplifies the business process and makes it very easy for both customers and the
+business. It ensures that the business retains more customers on a consistent basis and that clients
+can be reached out to and engaged with regularly. A business model like this makes it easy to
+anticipate demand and supply. Finally, this model offers much higher payment security for your
+business.
```

Comment:
Some repetition, I'd rewrite it 

Key point: Recurring revenue, ease of process for customers

some good ideas in this: 

For customers, the value lies in the convenience. First, there’s the autopilot simplicity of subscriptions that removes the thinking out of a purchase decision. Subscribers never have to remember to reorder every month, which gives them the reassurance that they will have whatever they need before they actually need it. Everything arrives to your door like magic -- removing the hassle of making a trip to the store or website to place an order. Second, subscriptions offer a flat rate which helps customers stay within their budget -- always. Lastly, subscriptions usually bring added value to the customer through bundling or getting it all for the price of one.

For businesses, the value of a subscription is the ability to predict revenue through recurring sales. In fact, according to John Warrillow, creator of The Value Builder System, recurring revenue is perhaps one of the most compelling factors in a company valuation. “The more guaranteed revenue you can offer a potential acquirer, the more valuable your business is going to be," Warrillow says. "Because a high percentage of the revenue of a subscription-based business is recurring, its value will be up to eight times that of a comparable business with very little recurring revenue.”